MSProject/ViewControllers/DetailBook/View/WBDetailBookHeaderView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  WBDetailBookHeaderView.swift
+//  MSProject
+//
+//  Created by Matias Spinelli on 13/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+protocol DetailBookDelegate {
+    func addToWishlist()
+    func rentBook()
+}
+
+class WBDetailBookHeaderView: UIView, NibLoadable {
+
+    @IBOutlet weak var bookImage: UIImageView!
+    @IBOutlet weak var bookTitle: UILabel!
+    @IBOutlet weak var bookAvailable: UILabel!
+    @IBOutlet weak var bookAuthor: UILabel!
+    @IBOutlet weak var bookYear: UILabel!
+    @IBOutlet weak var bookGenre: UILabel!
+    
+    @IBOutlet weak var customBackgroundView: UIView!
+
+    @IBOutlet weak var wishlistButton: WBButton!
+    @IBOutlet weak var rentButton: WBButton!
+    
+    var delegate: DetailBookDelegate?
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        backgroundColor = .clear
+    }
+    
+    var bookViewModel: WBBookViewModel? {
```

Comment:
The View shouldn't have an attribute referencing a ViewModel. Create a function to set all of these things and call it from the ViewController, maybe passing a `WBBookViewModel` as parameter.

Reply:
I think this is not wrong, but yeah, we usually use a function called setup(with: ViewModel) or something like that